internal/pkg/term/prompt/select_test.go

```diff
@@ -0,0 +1,170 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package prompt
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/stretchr/testify/require"
+)
+
+func TestOption_String(t *testing.T) {
+	testCases := map[string]struct {
+		input  Option
+		wanted string
+	}{
+		"should render the value with a tab if there is no hint": {
+			input: Option{
+				Value: "Help me decide!",
+			},
+			wanted: "Help me decide!\t",
```

Comment:
ohhh one small question! I know this is done intentionally but I just wonder why we want the extra `\t` in the end?